packages/eds-core-react/src/components/Typography/Typography.stories.tsx

```diff
@@ -1,220 +1,80 @@
-            label
-          </Typography>
-        </List.Item>
-        <List.Item>
-          <Typography group="navigation" variant="drawer_active">
-            drawer_active
-          </Typography>
-        </List.Item>
-        <List.Item>
-          <Typography group="navigation" variant="drawer_inactive">
-            drawer_inactive
-          </Typography>
-        </List.Item>
-        <List.Item>
-          <Typography group="navigation" variant="button">
-            button
-          </Typography>
-        </List.Item>
-        <List.Item>
-          <Typography group="navigation" variant="breadcrumb">
-            breadcrumb
-          </Typography>
-        </List.Item>
-        <List.Item>
-          <Typography group="navigation" variant="breadcrumb_hover">
-            breadcrumb_hover
-          </Typography>
-        </List.Item>
-      </List>
-    </List.Item>
-    <List.Item>
-      table
-      <List>
-        <List.Item>
-          <Typography group="table" variant="cell_header">
-            cell_header
-          </Typography>
-        </List.Item>
-        <List.Item>
-          <Typography group="table" variant="cell_text">
-            cell_text
-          </Typography>
-        </List.Item>
-        <List.Item>
-          <Typography group="table" variant="cell_text_bold">
-            cell_text_bold
-          </Typography>
-        </List.Item>
-        <List.Item>
-          <Typography group="table" variant="cell_text_link">
-            cell_text_link
-          </Typography>
-        </List.Item>
-        <List.Item>
-          <Typography group="table" variant="cell_numeric_monospaced">
-            cell_numeric_monospaced
-          </Typography>
-        </List.Item>
-      </List>
-    </List.Item>
-    <List.Item>
-      ui
-      <List>
-        <List.Item>
-          <Typography group="ui" variant="tooltip">
-            tooltip
-          </Typography>
-        </List.Item>
-        <List.Item>
-          <Typography group="ui" variant="snackbar">
-            snackbar
-          </Typography>
-        </List.Item>
-        <List.Item>
-          <Typography group="ui" variant="accordion_header">
-            accordion_header
-          </Typography>
-        </List.Item>
-        <List.Item>
-          <Typography group="ui" variant="chip__badge">
-            chip__badge
-          </Typography>
-        </List.Item>
-        <List.Item>
-          <Typography group="ui" variant="chart">
-            chart
-          </Typography>
-        </List.Item>
-      </List>
-    </List.Item>
-  </List>
-)
-GroupsOverview.parameters = {
-  docs: {
-    storyDescription:
-      'Variants ordered by their required group parameter. Example `<Typography group="input" variant="label"></Typography>`. Variants under `heading` and `paragraph` don’t require the group parameter and are therefore not included here.',
+export const GroupsOverview = () => {
+  return Object.entries(typography).map(([key, group]) => (
```

Comment:
This over shows the private property `_modes` (denoted by the _ in name). Suggest removing this using `.filter()` before `.map()`

Reply:
done